src/renderer/components/uielements/alert/ErrorAlert.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import { InfoCircleOutlined } from '@ant-design/icons'
+import { AlertProps } from 'antd/lib/alert'
+import * as A from 'fp-ts/lib/Array'
+import * as FP from 'fp-ts/lib/function'
+import * as O from 'fp-ts/lib/Option'
+
+import * as Styled from './ErrorAlert.style'
+
+type Props = Omit<AlertProps, 'description' | 'type' | 'showIcon' | 'icon'> & { descriptions?: string[] }
+
+const ErrorAlert: React.FC<Props> = (props: Props): JSX.Element => {
+  const { descriptions } = props
+
+  const description = FP.pipe(
+    descriptions,
```

Comment:
it seems we can just set default value for `descriptions` as an empty array. correct me if im wrong

Reply:
`descriptions` is optional, there is no default value.